vectorizer.py

```diff
@@ -65,11 +67,16 @@ async def vectorize(self, text: str, config: VectorInputConfig):
                 end_index = start_index + MAX_BATCH_SIZE
 
                 tokens = self.tokenize(sentences[start_index:end_index])
+                if self.mps:
+                    tokens.to("mps")
                 if self.cuda:
                     tokens.to(self.cuda_core)
                 batch_results = self.get_batch_results(tokens, sentences[start_index:end_index])
                 batch_sum_vectors += self.pool_embedding(batch_results, tokens, config)
-            return batch_sum_vectors.detach() / num_sentences
+                if True in torch.isnan(batch_sum_vectors).tolist():
```

Comment:
This is a bandaid to contain the explosion. Python still crashes and so benchmarks couldn't really be made.

Reply:
It is strange that the output of the NN contains `NaN`s, maybe we can print the `input` and `output` just to make sure where the error comes from.

Reply:
Also it can be reduced to this:
```python
if any(torch.isnan(batch_sum_vectors)):
  ...